packages/components/src/menu-group/index.js

```diff
@@ -19,6 +19,10 @@ export function MenuGroup( {
 	className = '',
 	instanceId,
 	label,
+	role = 'menu',
+	labelRole = ( role === 'group' ) ? { role: 'presentation' } : {},
+	useEventToOffset = true,
```

Comment:
We already have an eventToOffset to offset property, any reason for using a different name here  (useEventToOffset)?

Reply:
@jorgefilipecosta I felt it was clearer in this context, but if we prefer consistency, I can change that.